content-src/lib/selectLayoutRender.js

```diff
@@ -13,6 +16,11 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
         spocs.data.spocs && spocs.data.spocs.length) {
       const recommendations = [...data.recommendations];
       for (let position of spocsConfig.positions) {
+        const spoc = spocs.data.spocs[spocIndex];
+        if (!spoc) {
+          break;
```

Comment:
What's the purpose of this break?

Reply:
If there is no SPOCS left, I figured that we can break out from this for loop. Does it make sense?

Reply:
Ah ok I think I get it.

After walking through it, I think it's safe.

The spocIndex starts at 0 and just goes up. If at any point the spoc array has nothing from a 0 or greater, it's spent and there are no more spocs to even attempt to calculate.